### ISSUE TITLE
Average pool earnings fluctuating massively from long MRR rentals

### ISSUE THREAD

**Issue by samdave123** [RainbowMiner/RainbowMiner#1134]
My average earnings value spikes massively when MRR pays out and then slowly dwindles again, which is reducing the usefulness of the value to determine how much I've earnt. It seems it's being caused by a very small number of very large payments coming through from MRR.

Suggestion: Allow users to modify the number of day variable for average earnings so it can be increased to 2 weeks or even a month.

**Comment by RainbowMiner** [comment]
Yes, you are right. It creates a spike, due to that larger amount of crypto being paid. Good suggestion. I'll add this.

**Comment by samdave123** [comment]
My earnings just jumped from £4 to £19 after a 3 day rental ended. MRR earnings needs to be smoothed out or a longer period used.
